src/style.css

```
@@ -1 +1,65 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  position: relative;
+  display: flex;
+  padding: 0 50px;
+}
+
+.header__logo {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 10px 0;
+}
+
+.nav {
+  margin-left: auto;
+  display: inline-block;
+}
+
+.nav__wrap {
+  display: inline-block;
+  margin: 0;
+}
```

```suggestion
.nav__wrap {
  display: inline-block;
  margin: 0;
  padding: 0;
}
```
ul has default left padding